### DISCUSSION TITLE
FNIRSI GC-01: Battery discharge time

### DISCUSSION THREAD

**Discussion by tahinchik** [Gissio/radpro#172] [Q&A]
Dear Gissio, thanks for your time to develop this product.
Hello guys. Want to notify and ask about battery discharge from 100% to poweroff.
In "radpro-2.0.3" zip file i found "batterydischarge-data.csv" file and there device worked on one charge from 2023-06-11T17:25:05 to 2023-06-14T20:11:35. Thats more then 3 days... Dear @Gissio , is this data related to GC-01 or another device?
My GC-01 works approximately 12 hour with turn off display.....its terrible...the same time on default firmware from Fnirsi (but here display always ON). When i installed radpro i hoped that display OFF ability increase lifetime but this is not happened.
Want to ask all of you, how much time you device still work from 100 to 0 % ?
Thanks

**Comment by Gissio** [comment]
It's for the FS2011.

**Comment by ot4eto78** [comment]
> Dear Gissio, thanks for your time to develop this product. Hello guys. Want to notify and ask about battery discharge from 100% to poweroff. In "radpro-2.0.3" zip file i found "batterydischarge-data.csv" file and there device worked on one charge from 2023-06-11T17:25:05 to 2023-06-14T20:11:35. Thats more then 3 days... Dear @Gissio , is this data related to GC-01 or another device? My GC-01 works approximately 12 hour with turn off display.....its terrible...the same time on default firmware from Fnirsi (but here display always ON). When i installed radpro i hoped that display OFF ability increase lifetime but this is not happened. Want to ask all of you, how much time you device still work from 100 to 0 % ? Thanks

On my device, the battery drains in about 7-9 hours.
